src/simplejavatexteditor/SimpleJavaTextEditor.java

```diff
@@ -32,6 +34,13 @@ public class SimpleJavaTextEditor extends JTextPane {
      * @param args
      */
     public static void main(String[] args) {
+        //UIMANAGER
```

Comment:
I think this comment is quite redundant 🙃  as we can see in the try block we use UIManager (unless you had something else in mind you wanted to let know in the comment..)